doc/releases/changelog-dev.md

```diff
@@ -4,6 +4,34 @@
 
 <h3>New features since last release</h3>
 
+* New `qml.commutator` function that allows to compute commutators between
+  `qml.operation.Operator`, `qml.pauli.PauliWord` and `qml.pauli.PauliSentence` instances.
+  ```pycon
+  >>> qml.commutator(qml.PauliX(0), qml.PauliY(0))
+  2j*(PauliZ(wires=[0]))
+  ```
+
+  We can return a :class:`~PauliSentence` instance by setting `pauli=True`.
```

Comment:
Note that this linking with `:class` won't work in the version of the release notes on GitHub, so probably best just to remove.

Reply:
Thanks! Always forget about that in the markdown notes!